### DISCUSSION TITLE
So JP Primehome's streams golden era ended!

### DISCUSSION THREAD

**Discussion by letterfist** [iptv-org/community#1154] [Show and tell]
When [cdns.jp-primehome.com:8000](url) has been ended in this year after september lasting streaming climax airng, then nobody renewal new link anymore

**Comment by roxasmason274** [comment]
It was cool while it lasted

**Comment by jasonnguyen2606** [comment]
rest in peace, primehome playlist. it been a great run.

**Comment by BellezaEmporium** [comment]
![image](https://github.com/user-attachments/assets/4ac54121-f980-4a04-b338-3ff4e1b58b4f)

Wonder why you say that.

**Reply by roxasmason274** [reply]
What channel is that? Because most of the channels don't work anymore

**Reply by jasonnguyen2606** [reply]
> ![image](https://private-user-images.githubusercontent.com/30985701/391358215-4ac54121-f980-4a04-b338-3ff4e1b58b4f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzMwODQ0MzMsIm5iZiI6MTczMzA4NDEzMywicGF0aCI6Ii8zMDk4NTcwMS8zOTEzNTgyMTUtNGFjNTQxMjEtZjk4MC00YTA0LWIzMzgtM2ZmNGUxYjU4YjRmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMDElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjAxVDIwMTUzM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTYzMzkyNThhNTg4M2JiZGZlOTc2NDE2NGIzMDQyYzZjYzdkY2IyOTlhZDI0YThjYjg3NTY1MmQxY2FmZTUwZGYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.ci4CxctTzsT7-bTKbW68-cVupZzhArlUiRkl94OFUMQ)
> 
> Wonder why you say that.

How is this working already? I’m so confused.

**Reply by roxasmason274** [reply]
Not all the major channels are working I've seen

**Reply by BellezaEmporium** [reply]
> What channel is that? Because most of the channels don't work anymore

CS29 Ginga Channel

**Reply by jasonnguyen2606** [reply]
> ![image](https://private-user-images.githubusercontent.com/30985701/391358215-4ac54121-f980-4a04-b338-3ff4e1b58b4f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzMwODY0MTksIm5iZiI6MTczMzA4NjExOSwicGF0aCI6Ii8zMDk4NTcwMS8zOTEzNTgyMTUtNGFjNTQxMjEtZjk4MC00YTA0LWIzMzgtM2ZmNGUxYjU4YjRmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMDElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjAxVDIwNDgzOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTRiODA3MjE0Zjc0NzEzNWJmYmZhZGZmMTcwOGU2NjIxYTA1NjE5MjVjNGRhNWQ1OTQwZDkzMzk1ZTM0OTgwZTMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.RLqPn4AELZMqqf-IFTwSdeD9WA-PqGXqKLRGkM7teek)
> 
> Wonder why you say that.

It only works when you use your proxy ip and your port number wasn’t it?

**Reply by BellezaEmporium** [reply]
I use neither of those. Those are servers that jp-primehome seems to use.

**Reply by jasonnguyen2606** [reply]
Well what server was that?

**Reply by BellezaEmporium** [reply]
If I give the solution, the channels will be out within a few days. The only clue I can give is : keep fiddling with their available ISPs. And if you manage to find it, keep it up to yourself.

**Reply by jasonnguyen2606** [reply]
What does it mean? I don't understand this

**Reply by BellezaEmporium** [reply]
?isp=x on the original JP-Primehome URLs. this is what i'm talking about.

**Reply by Chaewonblight** [reply]
Will you share privately, please? (swear I won't share this link publicly, i only need to watch 3 music channels)

> ?isp=x on the original JP-Primehome URLs. this is what i'm talking about.

**Reply by BellezaEmporium** [reply]
> Will you share privately, please? (swear I won't share this link publicly, i only need to watch 3 music channels)
> 
> > ?isp=x on the original JP-Primehome URLs. this is what i'm talking about.
> 
> 

One clue I can tell you without telling you everything is to check the servers in which Primehome connects to. *Not all hosts are included in the ISP parameters.*

A specific header is the key to your research.

**Reply by yuyoiuyoiu** [reply]
> > Will you share privately, please? (swear I won't share this link publicly, i only need to watch 3 music channels)
> > > ?isp=x on the original JP-Primehome URLs. this is what i'm talking about.
> 
> One clue I can tell you without telling you everything is to check the servers in which Primehome connects to. _Not all hosts are included in the ISP parameters._
> 
> A specific header is the key to your research.

Sorry to bother you, do you know how long the sign= parameter in Primehome links stays valid?

**Reply by BellezaEmporium** [reply]
> > > Will you share privately, please? (swear I won't share this link publicly, i only need to watch 3 music channels)
> > > > ?isp=x on the original JP-Primehome URLs. this is what i'm talking about.
> > 
> > 
> > One clue I can tell you without telling you everything is to check the servers in which Primehome connects to. _Not all hosts are included in the ISP parameters._
> > A specific header is the key to your research.
> 
> Sorry to bother you, do you know how long the sign= parameter in Primehome links stays valid?

We haven't had to use the "sign" parameter before, so this might be something new. Depends on the type of token we're talking about, can be an hour or more.
